common/api/endpoint/ApiEndPoint.ts

```diff
@@ -7,6 +7,10 @@ export const ApiEndPoint = {
   },
   Assessment: {
     sections: (name = ':name', cycleName = ':cycleName') => apiPath('assessment', name, 'sections', cycleName),
+    OriginalDataPoint: {
+      one: (name = ':name', cycleName = ':cycleName', odpId = ':odpId') =>
+        apiPath('assessment', name, 'originalDataPoint', cycleName, odpId),
```

Comment:
path should be /assessments/:assessmentName/cycles/:cycleName/originalDataPoints/:originalDataPointId
that is the convention. should we change all the api paths in a separate pr ?

Reply:
ok